test/functional/api/command_spec.lua

```diff
@@ -782,6 +782,10 @@ describe('nvim_create_user_command', function()
     eq(5, #api.nvim_list_tabpages())
     eq(1, fn.tabpagenr())
   end)
+  it('addr option allows ranges without explicit range option', function()
```

Comment:
nit: each it() is separated by a blank line
```suggestion

  it('"addr" flag allows ranges without explicit "range" flag', function()
```